mcs/class/corlib/Makefile

```diff
@@ -298,11 +298,8 @@ NO_VTS_TEST = yes
 endif
 
 ifndef NO_VTS_TEST
-test: test-vts
-run-test: run-test-vts
-else
-test:
-run-test:
+test-local: test-vts
+run-test-local: run-test-vts
```

Comment:
this is the important change, the other changes are just for consistency/cosmetic.